frontend/src/app/components/user/profile/profile.component.html

```diff
@@ -12,6 +13,10 @@
     {{ currentContent.title }}
   </div>
   <div class="content-wrapper">
+    <div class="header">
```

Comment:
I would leave this like it was before but put the home button next to it or put the username etc on the right side of the header and the button on the left side, something like this but both home and username in the header